docs/Makefile

```diff
@@ -2,11 +2,14 @@
 
 docs:
 	docker pull grafana/docs-base:latest
-	docker run -v $(PWD)/sources:/hugo/content/docs/grafana/latest -p 3002:3002 --rm -it grafana/docs-base:latest /bin/bash -c 'npm i && make webpack && make docs-menu && hugo server -p 3002 -D --ignoreCache --baseUrl http://localhost:3002 --bind 0.0.0.0'
+	docker run -v $(PWD)/sources:/hugo/content/docs/grafana/latest -p 3002:3002 --rm -it grafana/docs-base:latest /bin/bash -c 'mkdir -p content/docs/grafana/latest/ && touch content/docs/grafana/latest/menu.yaml && make docs-menu && hugo server -p 3002 -D --ignoreCache --baseUrl http://localhost:3002 --bind 0.0.0.0'
 
 docs-no-pull:
-	docker run -v $(PWD)/sources:/hugo/content/docs/grafana/latest -p 3002:3002 --rm -it grafana/docs-base:latest /bin/bash -c 'make docs-menu && hugo server -p 3002 -D --ignoreCache --baseUrl http://localhost:3002 --bind 0.0.0.0'
+	docker run -v $(PWD)/sources:/hugo/content/docs/grafana/latest -p 3002:3002 --rm -it grafana/docs-base:latest /bin/bash -c 'mkdir -p content/docs/grafana/latest/ && touch content/docs/grafana/latest/menu.yaml && make docs-menu && hugo server -p 3002 -D --ignoreCache --baseUrl http://localhost:3002 --bind 0.0.0.0'
 
 docs-test:
 	docker pull grafana/docs-base:latest
 	docker run -v $(PWD)/sources:/hugo/content/docs/grafana/latest --rm -it grafana/docs-base:latest /bin/bash -c 'npm i && make prod'
+
+docs-windows:
+	docker run -v c:\GitProjects\grafana\docs\sources:/hugo/content/docs/grafana/latest -p 3002:3002 --rm -it grafana/docs-base:latest /bin/bash -c "mkdir -p content/docs/grafana/latest/ && touch content/docs/grafana/latest/menu.yaml && make docs-menu && hugo server -p 3002 -D --ignoreCache --baseUrl http://localhost:3002 --bind 0.0.0.0"
```

Comment:
Won't this depend on the user having Grafana checked out in c:\GitProjects\grafana?

Reply:
Nope, that is what the docker image is for.

Reply:
For Windows, yes. Remember, we got that path by copying it from my personal C drive. That path has nothing to do with Docker.

In order for a Windows user to make this command work, they need to replace that path with the path for their own personal Grafana repo on their machine.

Reply:
We had to make two major changes to get this to work on my system:
* Change the single quotes to double quotes.
* Replace PWD with my path to the Grafana docs > sources folder.

Reply:
@robbymilo as @oddlittlebird says, c:\GitProjects\grafana\docs\sources gets mounted from the host machine into the Docker container. Therefore that directory has to be present when issuing the command. Compare to the Unix version of the command where you use `$(PWD)` in order to get the current directory automatically.

Isn't there a method on Windows for getting the current directory, similar to `$(PWD)` on Unix, so we don't hardcode the user's Grafana checkout path?